rb/lib/selenium/webdriver/remote/w3c_capabilities.rb

```diff
@@ -79,7 +79,7 @@ def firefox(opts = {})
             opts[:browser_version] = opts.delete(:version) if opts.key?(:version)
             opts[:platform_name] = opts.delete(:platform) if opts.key?(:platform)
 
-            new({browser_name: 'firefox'}.merge(opts))
+            new({browser_name: 'firefox', marionette: true}.merge(opts))
```

Comment:
For Se 3.x we shouldn't need to specify `marionette: true`? I'll see if I can figure out where this is getting used that it might be necessary here.